public/sass/base/font-awesome/_larger.scss

```diff
@@ -11,11 +11,11 @@
   font-size: 2em !important;
 }
 .#{$fa-css-prefix}-3x {
-  font-size: 3em;
+  font-size: 3em !important;
```

Comment:
Could you please revert this change? I'm not sure what its suppose to solve. It doesn't seem to be related to this issue.

Reply:
You are right, this change is not related to the issue. I noticed it incidentally. Without the `!important`, the 3x, 4x and 5x classes don't have any effect. I will remove the commit for now.